tests/unit/test_merger_arxiv2arxiv.py

```diff
@@ -3060,23 +3060,7 @@ def test_documents():
         ]
     }
 
-    expected_merged = {
-        'documents': [
-            {
-                'key': 'pdf.pdf',
-                'description': 'paper',
-                'source': 'arXiv',
-                'fulltext': True,
```

Comment:
This one had the `fulltext` key, is that ok?

Reply:
yes, it's not a problem, we just take the update.